packages/plugins/graphql/server/src/services/builders/mutations/collection-type.ts

```diff
@@ -26,36 +25,35 @@ export default ({ strapi }: Context) => {
     const { uid } = contentType;
 
     const createMutationName = getCreateMutationTypeName(contentType);
-    const responseTypeName = getEntityResponseName(contentType);
+    const typeName = getTypeName(contentType);
 
     t.field(createMutationName, {
-      type: responseTypeName,
+      type: typeName,
+
+      extensions: {
```

Comment:
You've added this in a lot of definitions, could you just explain how/why it is used?

Reply:
Yes the i18N plugin was loking othe the type which can be a nexus type sometimes and doesn't match with the actual Strpai type. I decided to go with a strapi extensions  so we have a unified way to provide metadata for extensions